config/factorzoo.yaml

```diff
@@ -9,22 +9,48 @@ exchangeStrategies:
 - on: binance
   factorzoo:
     symbol: BTCUSDT
-    interval: 12h # T:20/12h
-    quantity: 0.95
+    linear:
+      enabled: true
+      interval: 1d
+      quantity: 1
+#      amount: 50_000
+      window: 5
+
+    exits:
+      - trailingStop:
+          callbackRate: 5%
+          activationRatio: 1%
+          closePosition: 100%
+          minProfit: 10%
+          interval: 1m
+          side: buy
+      - trailingStop:
+          callbackRate: 5%
+          activationRatio: 1%
+          closePosition: 100%
+          minProfit: 10%
+          interval: 1m
+          side: sell
+
+## (0) roiStopLoss is the stop loss percentage of the position ROI (currently the price change)
+#      - roiStopLoss:
+#          percentage: 1.0%
+#
+## (1) roiTakeProfit is used to force taking profit by percentage of the position ROI (currently the price change)
+## force to take the profit ROI exceeded the percentage.
+#      - roiTakeProfit:
+#          percentage: 35%
```

Comment:
remove these unused comment? you can create a `localconfig` directory for local testing config,